frontend/readcollection.lua

```diff
@@ -88,11 +88,13 @@ function ReadCollection:isFileInCollection(file, collection_name)
     return self.coll[collection_name][file] and true or false
 end
 
-function ReadCollection:isFileInCollections(file)
-    file = ffiUtil.realpath(file) or file
-    for _, coll in pairs(self.coll) do
-        if coll[file] then
-            return true
+function ReadCollection:isFileInCollections(file, ignore_show_mark_setting)
+    if ignore_show_mark_setting or G_reader_settings:nilOrTrue("collection_show_mark") then
```

Comment:
Same comment as https://github.com/koreader/koreader/pull/12940#discussion_r1898069976 :)
But I don't see in this PR any `isFileInCollections()` call with this added second parameter (?).

Reply:
Currently it is called to check whether to draw the mark only, without the second arg.
Added the second arg for future possible calls.